bootstrap.php

```diff
@@ -80,16 +80,23 @@ function ( $parsed_args, $url ) {
 
 			// Bail early if the request is not to PayPal's v2 checkout API
 			if ( false === stripos( wp_parse_url( $url, PHP_URL_HOST ), 'paypal.com' ) ) {
+				error_log( "Request is not for PayPal, skipping filter." );
```

Comment:
We probably don't want this, since it would add an entry for every external request